packages/@ngtools/webpack/src/plugin.ts

```diff
@@ -232,6 +237,17 @@ export class AotPlugin implements Tapable {
     if (options.hasOwnProperty('locale')) {
       this._locale = options.locale;
     }
+    if (options.hasOwnProperty('missingTranslation')) {
+      const version = new SemVer(VERSION.full);
+      if (version.compare(new SemVer('4.2.0-beta.0')) < 0) {
```

Comment:
Instead of this, just check the version as a string. I think we might be doing something similar elsewhere in this package for compiler-cli.